src/applications/vaos/services/mocks/index.js

```diff
@@ -227,20 +227,21 @@ const responses = {
   //   return res.json(errors);
   // },
 
-  'GET /vaos/v2/appointments/:id': (req, res) => {
-    const appointments = {
-      data: requestsV2.data.concat(confirmedV2.data).concat(mockAppts),
-    };
-    const appointment = appointments.data.find(
-      appt => appt.id === req.params.id,
-    );
-    if (appointment.start) {
-      appointment.future = moment(appointment.start).isAfter(moment());
-    }
-    return res.json({
-      data: appointment,
-    });
-  },
+  // 'GET /vaos/v2/appointments/:id': (req, res) => {
+  //   const appointments = {
+  //     data: requestsV2.data.concat(confirmedV2.data).concat(mockAppts),
+  //   };
+  //   const appointment = appointments.data.find(
+  //     appt => appt.id === req.params.id,
+  //   );
+
+  //   if (appointment.start) {
+  //     appointment.future = moment(appointment.start).isAfter(moment());
+  //   }
+  //   return res.json({
+  //     data: appointment,
+  //   });
+  // },
```

Comment:
Hi @accbjt , Just curious if the Appointments team can uncomment this mock call. This mock call helps our team with local testing.

Reply:
@jenniemc we have the same mock that we are using for our automated tests and local testing. If you can integrate this with the other one so that the tests don’t break then it would be okay.